scripts/irods/controller.py

```diff
@@ -155,7 +155,7 @@ def reload_configuration(self):
             if (new_agent_factory_pid is None) or (new_agent_factory_pid == old_agent_factory_pid):
                 time.sleep(1)
                 loop_counter += 1
-                if loop_counter == 15:
+                if loop_counter == 60:
```

Comment:
Provide a way for this value to be changed. Being able to change this value is important for testing.

Reply:
Created #8039

Reply:
This isn't a blocker for this PR.